### PR TITLE
Fix/kiosk issues on fulfilment screen

### PR DESCRIPTION
**Pull request by Andrewingombe**
## Description
Worked on the following issues in Kiosk:

- "ONLY DIGITAL PAYMENT ACCEPTED" text on fulfilment screen needs to be bigger.
- Increase margin above "HUNGRY?" on fulfilment screen.
- Clicking x on pay at counter modal should only close the modal. This was tested and found that it was working accordingly, need to text further.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Changes and Fixes
- "ONLY DIGITAL PAYMENT ACCEPTED" text on fulfilment screen needs to be bigger.
- Increase margin above "HUNGRY?" on fulfilment screen.
- Clicking x on pay at counter modal should only close the modal. This was tested and found that it was working accordingly, need to text further.

## Screenshots 
(prefer all screen sizes images)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

Affected Apps
- [x] Store
    - [x] Kiosk

### CODE REVIEW

File: store/src/styles/components/kiosk.scss [hunk 1]
```diff
@@ -128,7 +128,7 @@
    top: 0;
    left: 0;
    width: 100%;
-   height: 100%;
+   height: fit-content;
```
Comment (FAsami) [inline store/src/styles/components/kiosk.scss:131]:
```suggestion
   height: 100%;
```
Making it fit-content it will add a vertical scroll

**Review by FAsami — CHANGES_REQUESTED** [review]
image

Vertical space is appearing.

File: store/src/styles/components/kiosk.scss (hunk 1 above)
Comment (Andrewingombe) [reply store/src/styles/components/kiosk.scss:131]: @FAsami Thanks for that. The task was to make the text on the image bigger, thought making the height of the image fit-content would make the "Only digital text bigger..", with the height 100%, the image on fulfilment screen becomes vertically compressed and causes the text to be smaller. How would you suggest we make the text bigger on the image without making veritical scroll?

Comment (Andrewingombe) [reply store/src/styles/components/kiosk.scss:131]: OK what you were saying has been noted. So I was getting the resolution wrong. Thanks, am committing your suggestion. @FAsami

**Review by shubhamz950 — APPROVED** [review]

**Pull request merged**